### PR TITLE
change usage script, add openshift usage

### PR DESCRIPTION
**Pull request by fila43**

### CODE REVIEW

**Comment by fila43** [comment]
[test]

File: 5.26-mod_fcgid/s2i/bin/usage [hunk 1]
```diff
@@ -10,12 +10,14 @@ fi
...
 This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
-To use it, install S2I: https://github.com/openshift/source-to-image
+There are multiple ways how to run the image, see documentation at:
+https://github.com/sclorg/s2i-perl-container/blob/master/5.26-mod_fcgid/README.md
 
-Sample invocation:
+To use it in Openshift, run::
 
-s2i build https://github.com/sclorg/s2i-perl-container.git --context-dir=5.26/test/sample-test-app/ ${NAMESPACE}/perl-526${SUFFIX} perl-sample-app
+oc new-app perl:5.26-~https://github.com/sclorg/s2i-perl-container.git --context-dir=5.26/test/sample-test-app/
```
Comment (ppisar) [inline 5.26-mod_fcgid/s2i/bin/usage:18]: Is the hyphen in "perl:5.26-" correct?

Comment (ppisar) [inline 5.26-mod_fcgid/s2i/bin/usage:18]: A documentation for 5.26-mod_fcgid should use --context-dir=5.26-mod_fcgid/...

File: 5.30-mod_fcgid/s2i/bin/usage [hunk 2]
```diff
@@ -10,12 +10,14 @@ fi
...
 cat <<EOF
 This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
-To use it, install S2I: https://github.com/openshift/source-to-image
+There are multiple ways how to run the image, see documentation at:
+https://github.com/sclorg/s2i-perl-container/blob/master/5.30-mod_fcgid/README.md
 
 Sample invocation:
 
-s2i build https://github.com/sclorg/s2i-perl-container.git --context-dir=5.30/test/sample-test-app/ ${NAMESPACE}/perl-530${SUFFIX} perl-sample-app
+oc new-app perl:5.30~https://github.com/sclorg/s2i-perl-container.git --context-dir=5.30/test/sample-test-app/
```
Comment (ppisar) [inline 5.30-mod_fcgid/s2i/bin/usage:18]: A documentation for 5.30-mod_fcgid should use --context-dir=5.30-mod_fcgid/...

**Comment by ppisar** [comment]
The patch replaces s2i usage with OpenShift usage. Please rephrase to commit message like that.
Also I have zero knowledge about OpenShift. Please consult with @phracek or @hhorak.

**Comment by fila43** [comment]
[test]

File: 5.26-mod_fcgid/s2i/bin/usage [hunk 3]
```diff
@@ -10,12 +10,14 @@ fi
...
 This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
-To use it, install S2I: https://github.com/openshift/source-to-image
+There are multiple ways how to run the image, see documentation at:
+https://github.com/sclorg/s2i-perl-container/blob/master/5.26-mod_fcgid/README.md
 
-Sample invocation:
+To use it in Openshift, run::
 
-s2i build https://github.com/sclorg/s2i-perl-container.git --context-dir=5.26/test/sample-test-app/ ${NAMESPACE}/perl-526${SUFFIX} perl-sample-app
+oc new-app perl:5.26~https://github.com/sclorg/s2i-perl-container.git --context-dir=5.26-mod_fcgid/test/sample-test-app/
```
Comment (phracek) [inline 5.26-mod_fcgid/s2i/bin/usage:18]:
```suggestion
oc new-app perl:${PERL_VERSION}~https://github.com/sclorg/s2i-perl-container.git --context-dir=${PERL_VERSION}-mod_fcgid/test/sample-test-app/
```

File: 5.26/s2i/bin/usage [hunk 4]
```diff
@@ -6,12 +6,13 @@ NAMESPACE=centos
 
 cat <<EOF
 This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
-To use it, install S2I: https://github.com/openshift/source-to-image
+There are multiple ways how to run the image, see documentation at:
+https://github.com/sclorg/s2i-perl-container/blob/master/5.26/README.md
```
Comment (phracek) [inline 5.26/s2i/bin/usage:10]:
```suggestion
https://github.com/sclorg/s2i-perl-container/blob/master/${PERL_VERSION}/README.md
```

File: 5.26/s2i/bin/usage [hunk 5]
```diff
@@ -6,12 +6,13 @@ NAMESPACE=centos
...
 
-Sample invocation:
-
-s2i build https://github.com/sclorg/s2i-perl-container.git --context-dir=5.26/test/sample-test-app/ ${NAMESPACE}/perl-526-${DISTRO}7 perl-sample-app
+To use it in Openshift, run:
+oc new-app perl:5.26~https://github.com/sclorg/s2i-perl-container.git --context-dir=5.26/test/sample-test-app/
```
Comment (phracek) [inline 5.26/s2i/bin/usage:13]:
```suggestion
oc new-app perl:${PERL_VERSION}~https://github.com/sclorg/s2i-perl-container.git --context-dir=${PERL_VERSION}/test/sample-test-app/
```

File: 5.30-mod_fcgid/s2i/bin/usage [hunk 6]
```diff
@@ -10,12 +10,14 @@ fi
 
 cat <<EOF
 This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
-To use it, install S2I: https://github.com/openshift/source-to-image
+There are multiple ways how to run the image, see documentation at:
+https://github.com/sclorg/s2i-perl-container/blob/master/5.30-mod_fcgid/README.md
```
Comment (phracek) [inline 5.30-mod_fcgid/s2i/bin/usage:14]:
```suggestion
https://github.com/sclorg/s2i-perl-container/blob/master/${PERL_VERSION}-mod_fcgid/README.md
```

File: 5.30-mod_fcgid/s2i/bin/usage [hunk 7]
```diff
@@ -10,12 +10,14 @@ fi
...
 
 Sample invocation:
 
-s2i build https://github.com/sclorg/s2i-perl-container.git --context-dir=5.30/test/sample-test-app/ ${NAMESPACE}/perl-530${SUFFIX} perl-sample-app
+oc new-app perl:5.30~https://github.com/sclorg/s2i-perl-container.git --context-dir=5.30-mod_fcgid/test/sample-test-app/
```
Comment (phracek) [inline 5.30-mod_fcgid/s2i/bin/usage:18]:
```suggestion
oc new-app perl:${PERL_VERSION}~https://github.com/sclorg/s2i-perl-container.git --context-dir=${PERL_VERSION}-mod_fcgid/test/sample-test-app/
```

File: 5.30/s2i/bin/usage [hunk 8]
```diff
@@ -6,12 +6,14 @@ NAMESPACE=centos
 
 cat <<EOF
 This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
-To use it, install S2I: https://github.com/openshift/source-to-image
+There are multiple ways how to run the image, see documentation at:
+https://github.com/sclorg/s2i-perl-container/blob/master/5.30/README.md
```
Comment (phracek) [inline 5.30/s2i/bin/usage:10]:
```suggestion
https://github.com/sclorg/s2i-perl-container/blob/master/${PERL_VERSION}/README.md
```

File: 5.30/s2i/bin/usage [hunk 9]
```diff
@@ -6,12 +6,14 @@ NAMESPACE=centos
...
 
-Sample invocation:
+To use it in Openshift, run:
 
-s2i build https://github.com/sclorg/s2i-perl-container.git --context-dir=5.30/test/sample-test-app/ ${NAMESPACE}/perl-530-${DISTRO}7 perl-sample-app
+oc new-app perl:5.30~https://github.com/sclorg/s2i-perl-container.git --context-dir=5.30/test/sample-test-app/
```
Comment (phracek) [inline 5.30/s2i/bin/usage:14]:
```suggestion
oc new-app perl:${PERL_VERSION}~https://github.com/sclorg/s2i-perl-container.git --context-dir=${PERL_VERSION}/test/sample-test-app/
```

File: 5.26-mod_fcgid/s2i/bin/usage [hunk 10]
```diff
@@ -10,12 +10,14 @@ fi
 
 cat <<EOF
 This is a S2I ${IMAGE_DESCRIPTION} ${DISTRO} base image:
-To use it, install S2I: https://github.com/openshift/source-to-image
+There are multiple ways how to run the image, see documentation at:
+https://github.com/sclorg/s2i-perl-container/blob/master/5.26-mod_fcgid/README.md
```
Comment (phracek) [inline 5.26-mod_fcgid/s2i/bin/usage:14]:
```suggestion
https://github.com/sclorg/s2i-perl-container/blob/master/${PERL_VERSION}-mod_fcgid/README.md
```

**Review by phracek — CHANGES_REQUESTED** [review]
Only some version replacement with an environment variable. Thanks for the PR!.

**Comment by fila43** [comment]
thanks for the review, now it should be fine

**Comment by ppisar** [comment]
Good.

**Comment by fila43** [comment]
[test]

**Pull request merged**